clients/fides-js/src/components/tcf/TcfVendors.tsx

```diff
@@ -307,7 +349,7 @@ const TcfVendors = ({
 
   const filteredVendors = useMemo(() => {
     const legalBasisFiltered =
-      activeLegalBasisOption.value === LegalBasisEnum.CONSENT
+      activeLegalBasisOption.value === LegalBasisEnum.CONSENT.toString()
```

Comment:
So it *was* working, but it wasn't doing exactly what we expected. Since the `.value` property was always being set to a `LegalBasisEnum` value, the comparison was always correct. However, the types we have in place for `Option` were `{ label: string, value: string }` and under-the-hood Typescript was flagging that `LegalBasisEnum` was not compatible with `string`, so it was a footgun waiting to happen if we ever relied on `value` being string-y

None of us noticed this though, because I'm sure our VSCode Typescript version is newer than what we're running in these projects (Typescript 4.9.5 was released in January 2023) and string enums have gotten a lot smarter since then and can now just implicitly convert to strings without any issue 👍 

So this is just me being overly defensive and also wanting to ensure that, were we to add strict Typescript checks to our build, that this wouldn't surprise us